symengine/cwrapper.cpp

```diff
@@ -410,6 +443,16 @@ int is_a_ComplexDouble(const basic c)
 {
     return is_a<ComplexDouble>(*(c->m));
 }
+#ifdef HAVE_SYMENGINE_MPFR
+int is_a_RealMPFR(const basic c)
+{
+    return is_a<RealMPFR>(*(c->m));
+}
+int is_a_ComplexMPC(const basic c)
+{
+    return is_a<ComplexMPC>(*(c->m));
```

Comment:
This should return false if `HAVE_SYMENGINE_MPC` is not defined. (Note: MPC not MPFR)